juntagrico/templates/registration/password_reset_complete_cust.html

```diff
@@ -0,0 +1,12 @@
+{% extends "no_menu_base.html" %}
+{% load i18n %}
+{% load juntagrico.config %}
+{% block allcontent %}
+    <div class="offset-md-3 col-md-6">
+        <div class="alert alert-success mb-3">
+            <p>{% translate "Your password has been set.  You may go ahead and log in now." %}</p>
```

Comment:
leaving this in english will probably lead to some errors when we create the gettext files
i would insert the german translations